tests/ui_tools/test_boxes.py

```diff
@@ -202,11 +209,13 @@ def test_keypress_ENTER(self, panel_search_box,
     def test_keypress_GO_BACK(self, panel_search_box, back_key):
         size = (20,)
         panel_search_box.panel_view.view.controller.editor_mode = True
+        panel_search_box.set_caption("Search Results")
```

Comment:
I wonder if we should use the 'right-triangle symbol' here as well?

Reply:
I think my concern with this is that this is associated with a stream-topic separation. I'm looking at safe unicode and wondering about ⛏ instead.

Reply:
This Unicode character (emoji) blends in with the background and is hard to distinguish.

![image](https://user-images.githubusercontent.com/43616959/81145108-3f1c3880-8f93-11ea-94dd-872ec7d318f4.png)

![image](https://user-images.githubusercontent.com/43616959/81145054-17c56b80-8f93-11ea-974e-d2244909a02a.png)

Reply:
I only see the non-color (non-emoji) version of this, which doesn't have the same problem - it's just a white pick on green-ish background; see:
http://www.unicode.org/emoji/charts/emoji-variants.html#26cf

I've explored whether this can be expressed using a variant selector / presentation sequence, but it doesn't seem to change the output for me, and I can't find clear examples of how this is used in python.

I'm going to remove the pick for now, and we can explore this separately if we want to, possibly after discussion on the stream.